pkg/minikube/registry/drvs/docker/docker.go

```diff
@@ -240,6 +243,35 @@ func checkDockerVersion(o string) registry.State {
 		Doc:              docURL + "#requirements"}
 }
 
+func checkDockerDesktopVersion() *registry.State {
```

Comment:
can we ensure not to call the docker version command two times ? (since it takes 2-3 seconds)

we had this CachedVersion mechanisim in the past, 

I recommend getting the version in a struct once that has everything, and we can get the fields from the struct.

something like this
https://github.com/medyagh/minikube/blob/4d631bab7e32635a90afbbe6a8677e675be7d69c/pkg/drivers/kic/oci/info.go#L47

Reply:
The command only takes 133ms on my machine
```
$ time docker version --format {{.Server.Platform.Name}}
Docker Desktop 4.16.0 (95345)

real	0m0.133s
user	0m0.054s
sys	0m0.036s
```

But can definitely implement using a cached version

Reply:
Done